code/renderers/react/src/docs/jsxDecorator.tsx

```diff
@@ -190,7 +190,8 @@ export const jsxDecorator = (
 
   useEffect(() => {
     if (!skip) {
-      channel.emit(SNIPPET_RENDERED, (context || {}).id, jsx);
+      const { id, args } = context || {};
+      channel.emit(SNIPPET_RENDERED, id, jsx, false, args);
```

Comment:
given the amount of arguments to this event, we should convert them to a single object argument instead.

Reply:
I was aiming for backwards compatibility here (I guess we could do that even with changing it to an object based argument).